docs/setup/upgrade.md

```diff
@@ -1,4 +1,4 @@
-# Upgradation
+# Upgrading
```

Comment:
why not use nouns as titles?

Reply:
Upgradation isn't typically considered a word in several dialects of English.  See this Stack Exchange post: https://english.stackexchange.com/questions/192187/upgradation-not-universally-accepted

Just to check, I also took a look at other CNCF projects' documentation on the subject, and most that have upgrade documentation seem to follow the title pattern of 'Upgrading X'.  Here's some of what I found:
- [Argo](https://argoproj.github.io/argo-cd/operator-manual/upgrading/overview/) - uses 'upgrading' in the navigation and URL
- [Contour](https://projectcontour.io/resources/upgrading/) - 'Upgrading Contour'
- [Kubernetes](https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/) 'Upgrading kubeadm clusters'
- [Linkerd](https://linkerd.io/2/tasks/upgrade/) - 'Upgrading Linkerd'
- [NATS](https://docs.nats.io/nats-server/nats_admin/upgrading_cluster) - 'Upgrading a Cluster'
- [Vitess](https://vitess.io/docs/user-guides/upgrading-vitess/) - 'Upgrading Vitess'

Maybe this should be 'Upgrading KubeEdge'?

Reply:
'Upgrading KubeEdge' sounds good, let's do it!
BTW, please update pr title like "Docs: fix grammar issues".

Reply:
fixed doc title in 2edef69

Changed PR title to "Docs: fix grammar issues"